wgpu-core/Cargo.toml

```diff
@@ -29,55 +29,48 @@ wgsl = ["naga/wgsl-in"]
 vulkan-portability = ["hal/vulkan"]
 
 [dependencies]
-arrayvec = "0.7"
-bitflags = "1.0"
+arrayvec.workspace = true
+bitflags.workspace = true
 bit-vec = "0.6"
```

Comment:
Any reason we wouldn't just bring all dependencies up to the workspace level and inherit them all?

Reply:
I'm just not sure that if should put dependencies that are only used in a certain crate into the workspace. There is no other reason, I can do it.